### PR TITLE
Use tempest container image

### PR DESCRIPTION
**Pull request by arxcruz**
This patch removes the literal dockerfile to build tempest to use the one being build by the openstack-k8s-operators-ci

### CODE REVIEW

**Comment by arxcruz** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-openstack-operator-master-openstack-operator-build-deploy-tempest

**Comment by viroel** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-openstack-operator-master-openstack-operator-build-deploy-tempest

**Comment by viroel** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-openstack-operator-master-openstack-operator-build-deploy-tempest

Tag is now available:
tools:openstack-operator-tempest-container  Feb 20, 2023, 3:03 PM

**Comment by viroel** [comment]
/ph-rehearse ack

this change affects only _pull-ci-openstack-k8s-operators-openstack-operator-master-openstack-operator-build-deploy-tempest_

Tempest results:
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/36566/rehearse-36566-pull-ci-openstack-k8s-operators-openstack-operator-master-openstack-operator-build-deploy-tempest/1627733864212336640/artifacts/openstack-operator-build-deploy-tempest/openstack-k8s-operators-tempest/artifacts/stestr_results.html

**Comment by viroel** [comment]
/lgtm

thanks Arx

File: ci-operator/config/openstack-k8s-operators/openstack-operator/openstack-k8s-operators-openstack-operator-master.yaml [hunk 1]
```diff
@@ -7,6 +7,10 @@ base_images:
     name: golangci-lint
     namespace: ci
     tag: v1.46.0
+  openstack-operator-ci-tempest:
```
Comment (viroel) [inline ci-operator/config/openstack-k8s-operators/openstack-operator/openstack-k8s-operators-openstack-operator-master.yaml:10]: Already sets the name of container that is expected in tempest step[1]

[1] https://github.com/openshift/release/blob/master/ci-operator/step-registry/openstack-k8s-operators/tempest/openstack-k8s-operators-tempest-ref.yaml#L3

**Review by viroel — APPROVED** [review]

**Pull request merged**